docs/content/concepts/apis/exporting-apis.md

```diff
@@ -534,3 +534,58 @@ widgets                example.kcp.io/v1alpha1   false        Widget
 ```
 
 Furthermore, you can use the `.status.boundResources` field to precisely identify which `APIResourceSchemas` have been imported.
+
+## Resource storage
+
+When defining a resource in an APIExport, its storage is set to `crd` by default. This means the export's consumers will be able to interact with that resource as-if it had regular CRD semantics: creating, getting, listing, updating, patching and deleting all happen local to the APIBinding's workspace. Consequently, objects of this resource "live" in this workspace, and are backed by their own distinct objects in the etcd store.
+
+```yaml
+apiVersion: apis.kcp.io/v1alpha2
+kind: APIExport
+metadata:
+  name: example.kcp.io
+spec:
+  resources:
+    - group: example.kcp.io
+      name: widgets
+      schema: v220801.widgets.example.kcp.io
+      # (1)
+      # storage:
+      #   crd: {}
+```
+
+1. The `storage` block with `crd` declaration below is implicit. No need to define it manually.
+
+
+### Virtual resources
+
+!!! warning
+    As of 0.29, this feature is of alpha-version quality. To use it, enable the `CachedAPIs` feature gate.
+
+An APIExport resource may be defined with `virtual` storage. This means the resource is provided by a [virtual workspace](../workspaces/virtual-workspaces.md), and is projected into APIBinding's workspace.
+
+```yaml
+apiVersion: apis.kcp.io/v1alpha2
+kind: APIExport
+metadata:
+  name: example.kcp.io
+spec:
+  resources:
+    - group: example.kcp.io
+      name: managedwidgets
+      schema: v220801.managedwidgets.example.kcp.io
+      storage:
+        virtual:
+          reference: # (1)
+            apiGroup: example.kcp.io
+            kind: ManagedWidgetsEndpointSlice
```

Comment:
Lets not overcomplicate this for now. Lets use CachedResourceEndpoint slice, and add note in the bottom that virtual resources machinery is generic, and while we use it CachedResources feature, one can implement its own {any}EndpointSlice and as long as they comply with EndpointSlice machinery - it would work. 


But lets leave this as foot note ^